exercises/change/change_test.exs

```diff
@@ -30,6 +30,18 @@ defmodule ChangeTest do
   end
 
   @tag :pending
+  test "generates change with the least number of coins" do
+    change = %{3 => 34, 100 => 0}
+    assert Change.generate(102, [3, 100]) == {:ok, change}
+  end
+
+  @tag :pending
+  test "generates another change with the least number of coins" do
```

Comment:
This test seems redundant given the last test covers a very similar case? Am I missing something?